internal/operands/common-templates/reconcile.go

```diff
@@ -54,6 +60,34 @@ func (c *commonTemplates) WatchTypes() []runtime.Object {
 	return nil
 }
 
+func (c *commonTemplates) PauseCRs(request *common.Request) error {
+	patch := []byte(`{"metadata":{"annotations":{"kubevirt.io/operator.paused": "true"}}}`)
+	var kubevirtCommonTemplatesBundles kvsspv1.KubevirtCommonTemplatesBundleList
+	err := request.Client.List(request.Context, &kubevirtCommonTemplatesBundles, &client.ListOptions{})
```

Comment:
This function could use `UnstructuredList` instead of using specific type. It only lists old CRs and calls Patch on them. Then the old operator API code does not need to be imported as a module.

As a side effect, it would make this function more generic, and you could replace all 4 `PauseCRs` functions with one. The single function could be defined in `internal/commmon` package, or even in `controllers/ssp_controller.go`.